api/models/graphite.go

```diff
@@ -117,14 +117,25 @@ type GraphiteTagDetailsValueResp struct {
 }
 
 type GraphiteTagFindSeries struct {
-	Expr []string `json:"expr" form:"expr"`
-	From int64    `json:"from" form:"from"`
+	Expr   []string `json:"expr" form:"expr"`
+	From   int64    `json:"from" form:"from"`
+	Format string   `json:"format" form:"format" binding:"Default(seriesjson)"`
```

Comment:
1) should it not be `series-json` ? per the docs? 
2) seems to be missing validation.
this should work:
```suggestion
	Format string   `json:"format" form:"format" binding:"In(,series-json,lastupdate-json);Default(series-json)"`
```

Reply:
1. Yes, I can change that.
2. I didn't want to add that validation since `format` is currently ignored if specified. Adding the validation means breaking queries that used to work.

Reply:
you sure? note that the validation allows an empty string. we use this mechanism elsewhere too.

Reply:
Empty string would allow it to not exist, right? But what about `format=json`? Wouldn't that error with the validation (but doesn't error currently).

Reply:
> Empty string would allow it to not exist, right?

I believe so, but better try it and confirm.

> But what about format=json? Wouldn't that error with the validation (but doesn't error currently).

yes, but no one should be doing that as the format parameter never existed before.
If I'm missing something, you could also just add "json" to the list of accepted values and add it to the switch statement.

Reply:
> no one should be doing that 

Sure, but they might be. I was erring on the side of not breaking existing set ups.

> you could also just add "json"

That was just an example value. If you prefer the validation, I can add it. It just might break someone. I'll defer to you.

Reply:
I don't think adding random undocumented and unsuppored parameters constitutes valid use of an api. if that breaks requests of people who were doing something that the spec did not specify, that's fair enough. we can always revise should anyone complain very loudly.
i just don't see why we should accommodate parameters that didn't exist before. so yeah let's add the validation.